core/os/os.cpp

```diff
@@ -373,8 +373,16 @@ OS::ScreenOrientation OS::get_screen_orientation() const {
 void OS::_ensure_data_dir() {
 
 	String dd = get_data_dir();
-	DirAccess *da = DirAccess::open(dd);
+	String current_dir;
```

Comment:
Cool, I just assigned them right away instead. I need the current_dir variable a few lines later anyway and also at the end of the function. Let me know if you think this is fine or if you'd like that changed.